rke/k8s_rke_system_images.go

```diff
@@ -5431,10 +5431,9 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
-		// Enabled in v2.6.0
-		"v1.19.14-rancher1-2": {
+		"v1.19.15-rancher1-2": {
```

Comment:
There's a missing comment on top of v1.19.15-rancher1-2, add `Enabled in v2.6.1`

Reply:
Due to the OOB release and not waiting for v2.6.1, this comment will need to be `Enabled out of band post v2.6.0`